catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/ExportCommand.java

```diff
@@ -110,15 +112,14 @@
     private JarSigner jarSigner = new JarSigner();
 
     @Reference
-    private StorageProvider storageProvider;
+    protected StorageProvider storageProvider;
 
     @Option(name = "--output", description = "Output file to export Metacards and contents into. Paths are absolute and must be in quotes. Will default to auto generated name inside of ddf.home", multiValued = false, required = false, aliases = {
             "-o"})
     String output = Paths.get(System.getProperty("ddf.home"), FILE_NAMER.get())
             .toString();
 
-    @Option(name = "--delete", required = true, aliases = {
-            "-d"}, multiValued = false, description = "Delete Metacards and content after export. EG: --delete=true or --delete=false")
+    @Option(name = "--delete", required = true, multiValued = false, description = "Delete Metacards and content after export. E.g., --delete=true or --delete=false")
```

Comment:
It's like quickbuilding... using an alias means its harder when your not on something with your setup (and won't remember what to put), its more confusing to new people who dont realize what aliased commands are doing, etc.

I absolutely prefer making every person to run this command type this out **every single time**.